ansible_collections/arista/avd/docs/release-notes/5.x.x.md

```diff
@@ -237,6 +237,17 @@ With AVD version 5.0.0 the default encapsulation used for EVPN iBGP peering betw
 
 See the [porting guide](../porting-guides/5.x.x.md#default-encapsulation-for-wan-evpn-ibgp-peerings-changed-to-path-selection) for details.
 
+#### Allow custom PTP profiles
```

Comment:
IMO we don't really need to release-note this. It is a fix like many others which are just listed under fixes.

Reply:
removed in next commit